wercker.yml

```diff
@@ -68,16 +68,17 @@ build:
         else
           echo "build is skipped ..."
         fi
-
-  - script:
-      name: NoErrorTest - XWiki
-      code: |
-        if [[ $SKIP_CI == 'false' ]]; then
-          echo "Command: ./.ci/wercker.sh no-error-xwiki"
-          ./.ci/wercker.sh no-error-xwiki
-        else
-          echo "build is skipped ..."
-        fi
+# Temporary disabled as result of https://github.com/checkstyle/checkstyle/issues/5812
+# The check should be restored after the XWiki will be patched.
```

Comment:
not sure why you disabled it completely
we benefit a lot using xwiki in regression testing.
Are you uncomfortable to make pull request to xwiki ? or just disabled to make all other fixes first ?
Just let me know I am eager to help.

Reply:
See https://github.com/checkstyle/checkstyle/pull/5857#issuecomment-392784899 on example on how to create a patch

Reply:
done